parlai/scripts/convert_data_to_parlai_format.py

```diff
@@ -64,41 +65,47 @@ def dump_data(opt):
     fw.close()
 
 
-class ConvertDataToParlaiFormat:
+def setup_args():
+    # Get command line arguments
+    parser = ParlaiParser()
+    parser.add_argument(
+        '-n',
+        '--num-examples',
+        default=-1,
+        type=int,
+        help='Total number of exs to convert, -1 to convert \
```

Comment:
nit: while you're hree, can you clean up these strings (make them multi-line strings like you see in other places).